### ISSUE TITLE
How to train on other dataset

### ISSUE THREAD

**Issue by wyp19930313** [jjery2243542/adaptive_voice_conversion#29]
Hello, I use this code to train on the Chinese dataset. The loss on the training set is: 
AE:[425993/2000000], loss_rec=0.21, loss_kl=0.27, lambda=1.0e+0. 
And I find that the loss hardly decreases.
Then, i used the data in the training set for voice conversion, and found that The result was terrible, How can I improve it?

**Comment by atravler** [comment]
I have same problem, did you solve it?

**Comment by EmreOzkose** [comment]
Hi, Any improvement on this issue?
